Makefile.mingw64.mk

```diff
@@ -89,23 +92,40 @@ build_zynaddsubfx:
 #
 
 apply_mruby_patches: fetch_zest
-	cd $(ZEST_PATH)/deps/mruby-dir-glob ; \
+ifneq ($(ZEST_COMMIT), DIRTY)
+	cd $(ZEST_PATH)/mruby ; \
 	git checkout -- . ; \
-	patch -N < $(TOP)/mruby-dir-glob-no-process.patch
+	patch -p1 -N < ../string-backtraces.diff
+endif
 
-	cd $(ZEST_PATH)/deps/mruby-io ; \
-	git checkout -- . ; \
-	patch -N < $(TOP)/mruby-io-libname.patch
 
-	cd $(ZEST_PATH)/mruby ; \
+	#this patch is to fix a conflict between mingw and libuv.
+	# the conflict is fixed in later versions of libuv, but mruby-zest uses an earlier version currently.
+	cd $(ZEST_PATH)/deps/libuv ; \
 	git checkout -- . ; \
-	patch -p1 -N < $(TOP)/mruby-float-patch.patch
+	patch -p1 -N < $(TOP)/0001-build-fix-build-failures-with-MinGW-new-headers.patch
+	
+	#these patches are obsolete, and no longer should be applied.	
+	#cd $(ZEST_PATH)/deps/mruby-dir-glob ; \
+	#git checkout -- . ; \
+	#patch -N < $(TOP)/mruby-dir-glob-no-process.patch
+
+	#cd $(ZEST_PATH)/deps/mruby-io ; \
+	#git checkout -- . ; \
+	#patch -N < $(TOP)/mruby-io-libname.patch
+
+	#cd $(ZEST_PATH)/mruby ; \
+	#git checkout -- . ; \
+	#patch -p1 -N < $(TOP)/mruby-float-patch.patch
+
+
 
 setup_zest: fetch_zest apply_mruby_patches setup_libuv
-	cd $(ZEST_PATH) ; \
-	ruby rebuild-fcache.rb
+#	cd $(ZEST_PATH) ; \
+#	ruby rebuild-fcache.rb
```

Comment:
I'm pretty sure that make windows in zest doesn't build the cache file, so I think this is still a required step.

Reply:
I tested it again without the "ruby rebuild-fcache.rb", and it compiled fine.

I then tested it adding it back in and it compiled fine as well.